client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -71,8 +73,22 @@ class ManageMenu extends PureComponent {
 		return allSingleSites ? '' : '/my';
 	}
 
+	getEditorUrl = postType => {
```

Comment:
Why are we redeclaring this function here instead of reusing `getEditorUrl` from `state/selectors`?

Reply:
Because it needs to run for each post type, and I think it's more appropriate to do so when needed rather than in `mapStateToProps`.
But then again, I suppose I could also do something like this as well:
```es6
// mapStateToProps
editorUrls: reduce( postTypes, ( editorUrls, postType ) => {
  editorUrls[ postType ] = editorUrl( state, siteId, null, postType );
  return editorUrls;
}, {} ),
```